### PR TITLE
Animate NewReactionButton

### PR DESCRIPTION
**Pull request by buoyad**
Adds a bounce-in effect to the hover state of `NewReactionButton`

- [x] performance

### CODE REVIEW

**Comment by buoyad** [comment]
@keybase/react-hackers this is ready!
out

File: shared/chat/conversation/messages/react-button/index.js [hunk 1]
```diff
@@ -34,33 +25,64 @@ export type Props = {|
...
   ordinal: Types.Ordinal,
-  style?: StylesCrossPlatform,
+  style?: Styles.StylesCrossPlatform,
 |}
 
-const ButtonBox = glamorous(ClickableBox)({
-  ...(isMobile
+let bounceIn, bounceOut
+if (!Styles.isMobile) {
+  const glamor = require('glamor')
```
[the remark this answers is not in the document]
Comment (buoyad) [reply shared/chat/conversation/messages/react-button/index.js]: @chrisnojima It depends on DOM, native throws:
```
Can't find variable: document
```

**Comment by buoyad** [comment]
@keybase/react-hackers bump

File: shared/chat/conversation/messages/react-button/delay-interval.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  _delayID: TimeoutID
+
+  running: boolean
+
+  constructor(intervalMS: number, delayMS: number) {
+    this._intervalMS = intervalMS
+    this._delayMS = delayMS
+  }
+  start(fcn: () => void) {
+    if (this.running) {
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/react-button/delay-interval.js]: should remove running and just have that be a function of _delayID || _intervalID

File: shared/chat/conversation/messages/react-button/index.js [hunk 3]
```diff
@@ -114,33 +137,44 @@ export class NewReactionButton extends React.Component<NewReactionButtonProps, N
...
     this._stopCycle()
   }
 
   render() {
-    const ContainerComp = this.props.showBorder ? ButtonBox : ClickableBox
     return (
-      <ContainerComp
-        {...(isMobile ? {onLongPress: this.props.onLongPress} : null)} // or else desktop will complain
+      <ButtonBox
+        {...(Styles.isMobile ? {onLongPress: this.props.onLongPress} : null)} // or else desktop will complain
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/react-button/index.js]: we should fix this in the component, we keep on having to do this

**Comment by buoyad** [comment]
OK @chrisnojima, addressed comments PTAL

**Pull request merged**